extension/src/plots/model/collect.ts

```diff
@@ -312,60 +342,131 @@ export const collectData = (output: PlotsOutput): DataAccumulator => {
   return acc
 }
 
-type ComparisonPlotsAcc = { path: string; revisions: ComparisonRevisionData }[]
+type ComparisonPlotsAcc = {
+  classDetails: ComparisonClassDetails
+  path: string
+  revisions: ComparisonRevisionData
+}[]
 
 type GetComparisonPlotImg = (
   img: ImagePlot,
   id: string,
   path: string
 ) => ComparisonPlotImg
 
+export const boundingBoxColors = [
+  '#ff3838',
+  '#ff9d97',
+  '#ff701f',
+  '#ffb21d',
+  '#cfd231',
+  '#48f90a',
+  '#92cc17',
+  '#3ddb86',
+  '#1a9334',
+  '#00d4bb',
+  '#2c99a8',
+  '#00c2ff',
+  '#344593',
+  '#6473ff',
+  '#0018ec',
+  '#8438ff',
+  '#520085',
+  '#cb38ff',
+  '#ff95c8',
+  '#ff37c7'
+]
```

Comment:
There is also the file `extension/src/model/status/color.ts` that has a lot of colors inside. We should probably try to merge these two list (trying to reuse the same colors as much as possible) or at least have them in the same file so that it is easier to find.

Reply:
I vote we keep the colors separate since I don't want certain class colors to feel like they correlate with an particular experiments. 

Good idea on having both color sets in a file though! I'll save this for a followup since that would involve touching some experiment files and this PR is already pretty big.

Reply:
Moved colors into `src/common/colors.ts` but maybe there is a better spot.